docs-chef-io/content/inspec/signing.md

```diff
@@ -17,13 +17,13 @@ This page documents the `inspec sign` command introduced in InSpec 5 and details
 
 ### What is a Signed Profile?
 
-A signed profile, or `.iaf` file, is an InSpec profile with a digital signature attests to its authenticity. Many Progress Chef authored profiles are available as signed profiles from 2022.
+A signed profile, or `.iaf` file, is an InSpec profile with a digital signature that attests to its authenticity. Starting in 2022, many Progress Chef authored profiles are available as signed profiles.
 
```

Comment:
Many Progress Chef authored profiles are available as signed profiles from 2022. - this is direct usage. Please check.